content/chef_compliance_phase.md

```diff
@@ -0,0 +1,431 @@
++++
+title = "Compliance Phase"
+draft = false
+
+aliases = ["/chef_compliance_phase.html"]
+
+[menu]
+  [menu.infra]
+    title = "Compliance Phase"
+    identifier = "chef_infra/features/chef_compliance_phase.md Compliance Phase"
+    parent = "chef_infra/features"
+    weight = 80
++++
+
+[\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/master/content/chef_compliance_phase.md)
+
+## Overview
+
+Compliance Phase enables Chef Infra to run Chef InSpec profiles as part of a client run, by downloading profiles from sources like Chef Automate, Chef Supermarket, or GitHub, and reporting the results to Chef Automate or saving the results locally. Compliance Phase has been implemented to be mostly compatible with [audit cookbook](https://github.com/chef-cookbooks/audit), so that users of the audit cookbook can remove it from the node's runlist and get the same behavior.
```

Comment:
```suggestion
Compliance Phase enables Chef Infra to run Chef InSpec profiles as part of a client run, by downloading profiles from sources like Chef Automate, Chef Supermarket, or GitHub, and reporting the results to Chef Automate or saving the results locally. Compliance Phase has been implemented to be mostly compatible with legacy [audit cookbook](https://github.com/chef-cookbooks/audit), allowing users to remove the audit cookbook from the run_list and instead set `node['audit']['compliance_phase'] = true` to enable the Compliance Phase.
```

Reply:
We might want to just create a migration section and remove the whole sentence at the bottom here that tries to explain the migration in condensed form